packages/vats/src/core/basic-behaviors.js

```diff
@@ -317,14 +285,15 @@ export const addBankAssets = async ({
   bldIssuerKit.resolve(bldKit);
 
   const assetAdmin = E(agoricNamesAdmin).lookupAdmin('vbankAsset');
+  // XXX interposes bootstrap between agoricNames and vat-bank
```

Comment:
@michaelfig: I'll be able to fix this one with the callback abstraction (a closureless bound method descriptor).

Reply:
Seems to me that the simpler fix is to just pass `assetAdmin` and not bother with a separate `updater`. The only risk would be that vat-bank abuses `.reserve()`.